org.lflang/src/org/lflang/validation/LFValidator.java

```diff
@@ -852,29 +852,22 @@ public void checkReaction(Reaction reaction) {
     // FIXME: improve error message.
     }
 
-    @Check(CheckType.FAST)
-    public void checkReactor(Reactor reactor) throws IOException {
```

Comment:
Why remove this check?

Reply:
I don't think it is removed. See line 900.